packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -59,6 +63,14 @@ export interface IColumnHeaderCellProps extends IColumnNameProps, IProps {
     isColumnSelected?: boolean;
 
     /**
+     * If true, the column `name` will be replaced with a fixed-height skeleton and the
+     * `resizeHandle` will not be rendered. If passing in additional children to this component, you
+     * will also want to conditionally apply the `.pt-skeleton` class where appropriate.
+     * @default false
+     */
+    loading?: boolean;
```

Comment:
Asking because new to all of this and not sure about the styles -- all of the other boolean properties here are seem to start with "is", this one doesn't. Is there a reason for this?